pyglmnet/pyglmnet.py

```diff
@@ -15,33 +15,39 @@ def _lmb(distr, beta0, beta, X, eta):
     return _mu(distr, z, eta)
 
 
-def _mu(distr, z, eta, return_grad=False):
+def _mu(distr, z, eta):
     """The non-linearity (inverse link)."""
     if distr in ['softplus', 'gamma']:
         qu = np.log1p(np.exp(z))
```

Comment:
sorry to nitpick, but can all these variables names be `mu` or `grad_mu` instead of `qu` or `grad_qu`. thanks!